src/main/resources/swagger/api-docs.yaml

```diff
@@ -119,6 +121,24 @@ paths:
           description: Internal Error
           schema:
             $ref: '#/definitions/ErrorReport'
+  '/api/notebooks/{googleProject}/{clusterName}':
+    get:
+      summary: Access Jupyter notebooks on a dataproc cluster
+      description: Serves as a proxy for the Jupyer notebook server running the given dataproc cluster managed by Leo.
+      operationId: proxyCluster
+      tags:
+        - notebooks
+      parameters:
+        - in: path
+          name: googleProject
+          description: googleProject
+          required: true
+          type: string
+        - in: path
+          name: clusterName
+          description: clusterName
+          required: true
+          type: string
```

Comment:
I didn't include a `responses` field because it's not well-defined -- it's whatever Jupyter wants to return. Luckily swagger doesn't complain.